src/algorithms.jl

```diff
@@ -2936,6 +2941,7 @@ for Alg in [
     :ROS34PW1b,
     :ROS34PW2,
     :ROS34PW3,
+    :ROS34PRw,
```

Comment:
It would be great if we could get a docstring for the method mentioning the paper (reference above), e.g., using the `@doc` macro from Julia base.